compiler/src/iree/compiler/Dialect/VM/Conversion/VMToEmitC/EmitCBuilders.cpp

```diff
@@ -7,11 +7,42 @@
 #include "iree/compiler/Dialect/VM/Conversion/VMToEmitC/EmitCBuilders.h"
 
 #include "llvm/ADT/ArrayRef.h"
-#include "mlir/Dialect/EmitC/IR/EmitC.h"
```

Comment:
Even though `mlir/Dialect/EmitC/IR/EmitC.h` is included in `EmitCBuilders.h`, I would keep this include as you create verbatim ops later.